### ISSUE TITLE
[accessibility] Filters button on Timelines tab has obscure label

### ISSUE THREAD

**Issue by jcsteh** [feditext/feditext#432]
**Describe the bug**
On the Timelines tab, the button to adjust filters has the accessibility label "timelines.display-filters.edit". While it's easy enough to figure out what this does, it's not a particularly friendly label and my guess is that this comes from a name in the code rather than a label intended to be encountered by users.

**To Reproduce**
1. Open Feditext.
2. Go to the Timelines tab.
3. Turn on VoiceOver.
4. Tap or flick to the filters button.

**Expected behavior**
VoiceOver should say: "Filters, button" or "Change filters, button" or similar.
Instead, it says timelines.display-filters.edit, button".

**(please complete the following information):**
 - Device: iPhone SE 3rd generation
 - OS: iOS 18 public beta
 - App Version 1.7.2

**Comment by VyrCossont** [comment]
Thanks for the report! You're correct; this is a missing or incorrectly spelled localization key.
